sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -1852,6 +1852,15 @@ object SQLConf {
       .createWithDefault(
         "org.apache.spark.sql.execution.streaming.state.HDFSBackedStateStoreProvider")
 
+  val STATE_STORE_MAINTENANCE_THREADS =
+    buildConf("spark.sql.streaming.stateStore.stateStoreMaintenanceThreads")
```

Comment:
nit: `numStateStoreMaintenanceThreads` to make clear what this configures.

Also, `NUM_STATE_STORE_MAINTENANCE_THREADS` for variable name, and `numStateStoreMaintenanceThreads` in method name.